### PR TITLE
Set up ESLint for post service

### LINKED ISSUE #3157: Configure ESLint for `posts`
`src/api/posts` needs to be configured to add the new `@senecacdot/eslint-config-telescope` to `devDependencies` and configure `ESLint` to actually work.

### PR DESCRIPTION
**Pull request by aserputov**
## Issue This PR Addresses

Fixes #3157 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [ ] **UI**: Change which improves UI

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/api/posts/package.json [hunk 1]
```diff
@@ -25,6 +28,8 @@
     "normalize-url": "6.1.0"
   },
   "devDependencies": {
-    "supertest": "6.1.6"
+    "@senecacdot/eslint-config-telescope": "1.0.0",
+    "supertest": "6.1.6",
+    "eslint": "7.32.0"
```
Comment (cindyledev) [inline src/api/posts/package.json:33]: These should be listed in alphabetical order

```
"@senecacdot/eslint-config-telescope": "1.0.0",
    "eslint": "7.32.0",
    "supertest": "6.1.6"
```

**Review by cindyledev — CHANGES_REQUESTED** [review]

**Review by TueeNguyen — APPROVED** [review]

File: src/api/posts/package.json (hunk 1 above)
Comment (aserputov) [reply src/api/posts/package.json]: Do I need to fix back the order ? ```
"@senecacdot/eslint-config-telescope": "1.0.0",
"eslint": "7.32.0",
 "supertest": "6.1.6" ```

Comment (aserputov) [reply src/api/posts/package.json]: Because it's a `@` ?

Comment (cindyledev) [reply src/api/posts/package.json]: Yes please

**Review by TueeNguyen — APPROVED** [review]

**Review by rclee91 — DISMISSED** [review]

**Comment by rclee91** [comment]
Make sure to rebase and resolve conflicts first please. Thanks!

**Comment by aserputov** [comment]
@cindyledev while working on this issue, I discovered that there is a big if we use `@senecacdot/eslint-config-telescope": "1.0.0"`. At first, I thought that problem was with `rules` in `.eslint`. The bug: 
```
[Definition for rule '@typescript-eslint/no-shadow' was not found](https://stackoverflow.com/questions/66033302/definition-for-rule-typescript-eslint-no-shadow-was-not-found 
```

**Comment by cindyledev** [comment]
According to #3247, we have chosen to unlink local packages so any `eslint-config-telescope` shouldn't be using `workspace:1.0.0`. It should be `1.0.0` instead

**Comment by aserputov** [comment]
@cindyledev ` 1:1  error  Definition for rule '@typescript-eslint/no-shadow' was not found  @typescript-eslint/no-shadow`. Do you have any guess, why this can be? I think it's not only on my side.

File: src/api/posts/package.json [hunk 2]
```diff
@@ -25,6 +28,8 @@
     "normalize-url": "6.1.0"
   },
   "devDependencies": {
+    "@senecacdot/eslint-config-telescope": "1.0.0",
```
Comment (cindyledev) [inline src/api/posts/package.json:31]: We just updated eslint-config-telescope to v1.1.0 in #3267 and many of the eslint configurations were refactored in #3253. To pick up the latest updates which addresses the no-shadow error, change the `1.0.0` to `1.1.0`

**Pull request closed without merging**